docs/resources/project_approval_rule.md

```diff
@@ -0,0 +1,47 @@
+# gitlab\_project\_approval\_rule
+
+This resource allows you to create and manage approval rules for your GitLab
+projects. For further information on approval rules, consult the [gitlab
+documentation](https://docs.gitlab.com/ee/api/merge_request_approvals.html#project-level-mr-approvals).
+
+*Note*: This feature requires a GitLab Starter account or above.
+
+## Example Usage
+
+```hcl
+resource "gitlab_project_approval_rule" "example" {
```

Comment:
What would this look like if the user wants multiple approval rules? Would they create multiple `gitlab_project_approval_rule` with the same project ID? Would that work? Could you add to the description for this doc that you can create multiple of these resources with the same project ID, since it's a bit uncommon to have resources that generate their own ID, not based on attributes?

Reply:
I added an extra approval rule to the examples. Thanks!